src/routes/index.ts

```diff
@@ -308,4 +309,10 @@ router.route(config.UPDATE_CHECK_YOUR_ANSWERS_URL)
   .get(updateCheckYourAnswers.get)
   .post(updateCheckYourAnswers.post);
 
+router.get(config.UPDATE_BENEFICIAL_OWNER_GOV_URL, authentication, navigation.hasBeneficialOwnersStatement, updateBeneficialOwnerGov.get);
```

Comment:
you can't use navigation.hasBeneficialOwnerStatement since it refers to Registration, you will have to add in a new navigation check or maybe it can be done in UAR-260, or 134